yarn.lock

```diff
@@ -0,0 +1,267 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We probably shouldn't have `yarn.lock` and `package-lock.json` hanging out together :)

I only went with the latter because I didn't have `yarn` installed on my personal machine, and don't have a strong preference between the two (is yarn still faster in 2023? idek), so if you want to keep this and delete the npm-generated lockfile, that's great with me!

Reply:
Not sure if it's faster, just my default manager, so I didn't think twice about it. Will remove it and update the npm one for consistency.